main.py

```diff
@@ -60,21 +70,35 @@ def save_tokns(tokens):
         json.dump(tokens, outfile)
 
 
-if __name__ == "__main__":
-    tokens = load_tokens()
-
-    data = get_data()
+def get_data_page(page=None):
+    data = get_data(page)
     data = parse_data(data)
     data = get_houses_list(data)
+    data = data[::-1]
+    return data
 
+
+def process_data(data, tokens):
     for house in data:
-        house_data = extract_each_house(house)
+        house_data = extract_house_data(house)
         if house_data is None:
             continue
-        if house_data['token'] in tokens:
+        if house_data["token"] in tokens:
             continue
 
-        tokens.append(house_data['token'])
+        tokens.append(house_data["token"])
         send_telegram_message(house_data)
+        time.sleep(1)
+    return tokens
+
+
+if __name__ == "__main__":
+    print(datetime.datetime.now())
+    tokens = load_tokens()
+    print(len(tokens))
```

Comment:
Please remove all the unnecessary `print`s. It's better to use `log` instead.

Reply:
agreed